cmd/lakefs/cmd/root.go

```diff
@@ -49,15 +49,15 @@ func init() {
 }
 
 func validateQuickstartEnv(cfg *config.Config) {
-	if !((cfg.Database.Type == local.DriverName || cfg.Database.Type == mem.DriverName) && cfg.Blockstore.Type != block.BlockstoreTypeLocal) {
-		fmt.Printf("quickstart mode can only run with local settings\n")
+	if !((cfg.Database.Type == local.DriverName || cfg.Database.Type == mem.DriverName) && cfg.Blockstore.Type == block.BlockstoreTypeLocal) {
```

Comment:
I think this is simpler.
```suggestion
	if ((cfg.Database.Type != local.DriverName && cfg.Database.Type != mem.DriverName) || cfg.Blockstore.Type != block.BlockstoreTypeLocal) {
```